README.md

```diff
@@ -60,17 +60,17 @@ There are roughly three types of TrueType tables:
 | Rendering         | -<sup>1</sup>          | ✓                   | ~ (very primitive)             |
 | `ankr` table      | ✓                      |                     |                                |
 | `avar` table      | ✓                      | ✓                   |                                |
-| `bdat` table      |                        | ✓                   |                                |
-| `bloc` table      |                        | ✓                   |                                |
-| `CBDT` table      | ✓                      | ✓                   |                                |
+| `bdat` table      | ~ (no 4)               | ✓                   |                                |
+| `bloc` table      | ✓                      | ✓                   |                                |
+| `CBDT` table      | ~ (no 8, 9)            | ✓                   |                                |
```

Comment:
I guess no 4 here as well?

Reply:
I don't think so. The Microsoft specs say that it's only supported by Apple, but the Apple specs use bdat+bloc instead of EBDT+EBLC or CBDT+CBLC.

Reply:
Ugh... ok. AAT and OT are indeed sometimes disagree on stuff.